src/main/java/com/deavensoft/paketomat/center/model/Paketomat.java

```diff
@@ -15,14 +16,24 @@ public class Paketomat {
     @GeneratedValue(strategy = GenerationType.AUTO)
     @Column(name = "id")
     private Long id;
-    @JsonIgnore
-    @ManyToOne
-    @JoinColumn(name="city", nullable=false)
+    @ManyToOne()
+    @JoinColumn(name="city", nullable = false)
     private City city;
 
-    @Transient
+    @Column(name = "serial_number", unique = true)
+    public Long addr;
+    public static Long serialNumber = 1L;
```

Comment:
Make serialNumber a static final constant or non-public and provide accessors if needed.

Reply:
We changed it so that addr gets serialNumber++ value in constructor and both fields are now private

Reply:
Can you explain to me the logic behind the serialNumber and address? I don't understand the connection between them.

Reply:
Changed it